src/auslib/AUS.py

```diff
@@ -73,29 +73,35 @@ def evaluateRules(self, updateQuery, transaction=None):
         self.log.debug("Looking for rules that apply to:")
         self.log.debug(updateQuery)
 
+        eval_metadata = {}
```

Comment:
I think it would be good to set defaults for rule_id & rule_data_version here, for a couple of reasons:
1) To make it more obvious what's expected in this data structure
2) To guard against a caller that may try to look for one of the above even if they're unset.

Let's default them to `unknown`